### PR TITLE
Add customizable codePattern height + resizeable

### LINKED ISSUE #5988: Make the code area in CodePattern resizeable or scrollable.
Context: https://github.com/GoogleChrome/web.dev/pull/5970#discussion_r689824115

### PR DESCRIPTION
**Pull request by devnook**
Fixes #5988

There are 3 ways to define height of the element:

1. Add height param in CodePattern shortcode include : {% CodePattern patternId HEIGHT %}
2. Add height in the Pattern frontmatter
3. Default height calculated from the number of lines of the longest code snippet

The precedence is 1 > 2 > 3.

Th iframe is now also resizeable (see handle in bottom right corner of the iframe).

### CODE REVIEW

File: src/site/content/en/handbook/web-dev-components/index.md [hunk 1]
```diff
@@ -322,7 +322,12 @@ element, and add the `w-ml--l` class to the label. See below:
 
 ## Code pattern {: #codepattern }
 
-A component that displays a demo and code snippets, organized in tabs, side by side.
+A component that displays a demo and code snippets, organized in tabs,
+side by side.
+
+The height of the component is determined by the length of the longest code
+snippet. It can be overridden by passing the optional height attribute
+in the shortcode.
```
Comment (jpmedley) [inline src/site/content/en/handbook/web-dev-components/index.md:330]: Height is determined by length? Is this because of wrapping?

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/lib/components/Tabs/_styles.scss [hunk 2]
```diff
@@ -88,6 +88,7 @@ web-tabs {
     display: flex;
     flex: 1;
     flex-direction: column;
+    overflow-y: scroll;
```
Comment (samthor) [inline src/lib/components/Tabs/_styles.scss:91]: @khempenius has a PR which does this too. Why not `auto`, which will only optionally show the scrollbar?

File: src/site/content/en/handbook/web-dev-components/index.md (hunk 1 above)
Comment (samthor) [reply src/site/content/en/handbook/web-dev-components/index.md:330]: I think this means "code length" in the sense of, long code will cause lots of newlines.

File: src/site/_includes/components/CodePattern.js [hunk 3]
```diff
@@ -28,12 +53,16 @@ module.exports = (patternId, height = 300) => {
     })
     .join('\n');
 
+  const defaultHeight =
+    Math.max(...assetLines) * lineHeight + 2 * preMargin + tabListHeight;
```
Comment (samthor) [inline src/site/_includes/components/CodePattern.js:57]: can you move the "extra 2 lines" to the top of the file?

**Review by samthor — APPROVED** [review]

File: src/site/content/en/handbook/web-dev-components/index.md (hunk 1 above)
Comment (devnook) [reply src/site/content/en/handbook/web-dev-components/index.md:330]: @jpmedley Please unblock this PR by removing "requested changes" review

**Pull request merged**